src/server/context.ts

```diff
@@ -36,6 +37,9 @@ export async function getAppContext(
     // We want the originating client, which is the leftmost IP address
     // if x-forwarded-for is an array
     'origin-client-ip': extractHeader(req.headers['x-forwarded-for']),
+    'apollo-require-preflight': extractHeader(
+      req.headers['apollo-require-preflight']
+    ),
```

Comment:
Just like on lines 37-38 for the `origin-client-ip` header, could you please add a comment for this header explaining why it needed to be added? I think keeping that information alongside the code, too, is useful compared to specifying it in the PR description only.